packages/core/src/GenericBanner/GenericBanner.js

```diff
@@ -130,6 +129,13 @@ GenericBanner.propTypes = {
     ),
   ]),
   heading: PropTypes.node,
+  headingFontSize: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+    PropTypes.arrayOf(
+      PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+    ),
+  ]),
```

Comment:
Can we use the `fontSize` prop types like we do in [Text](https://github.com/priceline/design-system/blob/master/packages/core/src/Text/Text.js#L89)?